src/app/core/views/Preferences/tabs/Account/InviteAFriend.tsx

```diff
@@ -37,7 +37,7 @@ export default function InviteAFriend({ className = '' }: { className?: string }
       const castedError = errorService.castError(err);
       if (castedError.message !== 'Mail invitation daily limit reached') {
         notificationsService.show({
-          text: translate('error.inviteAFriend', { message: castedError.message }),
+          text: translate('error.inviteAFriendEmpty'),
```

Comment:
If `inviteAFriend` translation is not used any more, please, remove it

Reply:
It is used in another component